### DISCUSSION TITLE
升级到2.11.8后，隐藏图标，打不开设置（win10）界面，直接进josn文件，但打开图标显示，就能进设置页面了

### DISCUSSION THREAD

**Discussion by XDRSZYHRL** [shuax/MouseInc.Settings#205] [Q&A] [answered]
版本	Windows 10 专业工作站版
版本号	21H2
安装日期	‎2021/‎5/‎21
操作系统内部版本	19044.1415
体验	Windows Feature Experience Pack 120.2212.3920.0
隐藏图标指配置文件中的ShowTrayIcon选项，当该选项值为false时，无法进入 设置界面，只能进入josn文本编辑记事本，当该选项值为true时，可以进入设置界面

**Comment by shuax** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6940583/147202041-04f76ace-01cd-419f-9b54-89760d62b1af.png)
你隐藏的时候已经警告过了。

**Reply by XDRSZYHRL** [reply]
隐藏后，通过手势M打开设置时只能打开
![image](https://user-images.githubusercontent.com/59906163/147202273-e8151bc1-077d-4b77-b2c4-d94c0727b798.png)
而不能打开
![image](https://user-images.githubusercontent.com/59906163/147202331-80f7809b-ac34-4d0b-badf-0e79758e5850.png)
这个

**Comment by XDRSZYHRL** [comment]
现在发现，不管是否隐藏，使用手势M打开时都是
![image](https://user-images.githubusercontent.com/59906163/147202606-349cc685-9151-40f2-b9b2-8ee3f756bfa4.png)
这个

**Comment by shuax** [comment]
你改过M吧

**Comment by shuax** [comment]
![image](https://user-images.githubusercontent.com/6940583/147202983-8f9cc9b2-b211-4cca-8195-60bd85923919.png)
默认是这样

**Reply by XDRSZYHRL** [reply]
![image](https://user-images.githubusercontent.com/59906163/147203074-cb30a838-68e7-4e99-a3b0-3c178a1434e7.png)
我的

**Comment by XDRSZYHRL** [comment]
这是我的设置文件
[MouseInc.zip](https://github.com/shuax/MouseInc.Settings/files/7767421/MouseInc.zip)

**Comment by shuax** [comment]
你多勾了一个M啊

**Comment by XDRSZYHRL** [comment]
不好意思，麻烦了。可能是不小心勾上了
